model.py

```diff
@@ -1,17 +1,33 @@
 import torch
 from vllm import LLM, SamplingParams
-from config import CKPT_ID, PROMPT
+from config import (
+    LLAVA_PROMPT, QWEN_VL_PROMPT,
+    LLAVA_CKPT_ID, QWEN_VL_CKPT_ID,
+)
 
+CKPT_ID_MAP = {
+    "llava": LLAVA_CKPT_ID,
+    "qwen-vl": QWEN_VL_CKPT_ID,
+}
+PROMPT_MAP = {
+    "llava": LLAVA_PROMPT,
+    "qwen-vl": QWEN_VL_PROMPT,
+}
 
-def load_vllm_engine(max_tokens=120):
+def load_vllm_engine(model_name, max_tokens=120):
```

Comment:
We should default to a model_name (`llava` in this case), otherwise, this is breaking change.